components/gitpod-db/src/typeorm/entity/db-workspace-instance.ts

```diff
@@ -40,6 +40,12 @@ export class DBWorkspaceInstance implements WorkspaceInstance {
     })
     deployedTime?: string;
 
+    @Column({
+        default: '',
+        transformer: Transformer.MAP_EMPTY_STR_TO_UNDEFINED
+    })
+    stoppingTime?: string;
```

Comment:
a comment that explains what the difference between `stoppingTime` and `stoppedTime` is would help avoid missunderstandings.